bedrock/firefox/templates/firefox/whatsnew/index-account.html

```diff
@@ -6,7 +6,7 @@
 
 {% extends "firefox/whatsnew/base.html" %}
 
-{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_title %}{{ ftl('whatsnew-page-title-v2' ,fallback='whatsnew-page-title') }}{% endblock %}
```

Comment:
**Nit:** The space is on the wrong side of the comma (for all the WNP titles)

If fixing this fills you with dread we can totally leave it.

Reply:
> If fixing this fills you with dread we can totally leave it.

Well when you put it like that... 😝 j/k, I'll fix it